Makefile

```diff
@@ -205,9 +209,10 @@ help:
 go-check:
 	$(eval GO_VERSION := $(shell printf "%03d%03d%03d" $(shell $(GO) version | grep -Eo '[0-9]+\.[0-9.]+' | tr '.' ' ');))
 	@if [ "$(GO_VERSION)" -lt "$(MIN_GO_VERSION)" ]; then \
-		echo "Gitea requires Go 1.16 or greater to build. You can get it at https://golang.org/dl/"; \
+		echo "Gitea requires Go $(MIN_GO_VERSION_STR) or greater to build. You can get it at https://go.dev/dl/"; \
 		exit 1; \
 	fi
+	@echo "checked go matches min version: $(MIN_GO_VERSION_STR)"
```

Comment:
It should be silent on success, else we have this spam on every development target we run.

Reply:
It's not that spam, it just tell developers that the version check works. Fine to remove. Removed